source/Server.Contracts/Variables/ActionSpecialVariable.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+
+namespace Sashimi.Server.Contracts.Variables
+{
+    public class ActionSpecialVariable
```

Comment:
I'm not sure about the name. These could be Action or Account or a number of other things in the future, right?

Reply:
Happy to rename this. We also have [SpecialVariable](https://github.com/OctopusDeploy/OctopusDeploy/blob/977fc88619d82a2449dc69d934e42cd9d4b2e1f7/source/Octopus.Core/Variables/SpecialVariable.cs#L7) on the server-side that also have a `CustomBindingLabel`. We could call both `SpecialVariable` or make the server-side one `SpecialVariableDefinition`?